File: problemas/iniciante/2061/2061.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

let [N, M] = lines.shift().trim().split(' ');
for (let i = 0; i < M; ++i) {
    let acao = lines.shift().trim();

    if (acao === "fechou") {
        ++N;
    } else {
        --N;
    }
}

console.log(N);

File: problemas/iniciante/2061/2061.cs
using System;
using System.Collections.Generic;
using System.Linq;

class URI {
    static void Main(string[] args) {
        List<int> entradas = Console.ReadLine().Trim().Split(' ').Select(x => int.Parse(x)).ToList();
        int N = entradas[0];
        int M = entradas[1];

        for(int i = 0; i < M; ++i){
            string acao = Console.ReadLine();

            if(acao == "fechou"){
                ++N;
            }else{
                --N;
            }
        }

        Console.WriteLine(N);
    }
}